src/content/blog/2024-08-09-utah-sgid-statewide-roads-data-layer-updates.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Utah SGID Statewide Roads Data Layer Updates
+author: Greg Bunce
+date: 2024-08-09T08:30:00.000Z
+category: SGID Updates
+tags:
+  - roads
+cover_image: /src/images/pillar-blog/road-update-social-card.png
+cover_image_alt: ugrc sgid road update social card
+---
+
+UGRC recently completed updates to the SGID Roads data layer. Please visit our [Roads and Highway System](/products/sgid/transportation/road-centerlines/) data page where you will find information about the Roads data model, as well as a web service layer to the SGID Roads data and direct download links in shapefile and geodatabase format.
+
+These updates are also reflected in UGRC's [address locators](/products/sgid/address/).
+
+The following are highlights from this month's update.
+
+#### County Updates
+
+New roads were added and road names and address ranges were updated for the following counties:
+
+- **Davis County:** Obtained roads data on 07/15/2024. Previous update was on 06/14/2024.
+- **Salt Lake County:** Obtained roads data on 07/15/2024. Previous update was on 07/19/2024.
+- **Utah County:** Obtained roads data on 07/15/2024. Previous update was on 07/19/2024.
+- **Washington County:** Obtained roads data on 07/30/2024. Previous update was on 07/19/2024.
+- **Weber County:** Obtained roads data on 07/22/2024. Previous update was on 07/19/2024.
```

Comment:
Some of these dates look a little suspicious. For example, SLC data obtained on 7/15/24 but previous update was 7/19/2024? Would you like double-check them?

Reply:
I thought the same but it seems reasonable that we got new data after we did the last update.